### DISCUSSION TITLE
Can the spike time of each neuron be saved?

### DISCUSSION THREAD

**Discussion by xiaoxiaochaochao** [fangwei123456/SpikingFlow#187] [Q&A] [answered]
Hello,
fangwei, I'm wonder the spike time of neurons in a network can be saved?

**Comment by fangwei123456** [comment, marked as answer]
You can refer to this: https://github.com/fangwei123456/spikingjelly/issues/182#issuecomment-1062466323

**Reply by xiaoxiaochaochao** [reply]
Thank you!
